auditbeat/_meta/fields.common.yml

```diff
@@ -0,0 +1,36 @@
+- key: common
+  title: Common
+  description: >
+    Contains common fields available in all event types.
+  fields:
+
+    - name: metricset.module
```

Comment:
Yeah, those are present because it fully dependent on the Metricbeat framework. I think through the experimental/beta phase we should refine both the data model and config.